src/TLuaInterpreter.cpp

```diff
@@ -3203,92 +3203,212 @@ int TLuaInterpreter::hideUserWindow(lua_State* L)
+            break;
+        }
+        case 2: {
+            if (!lua_isnumber(L, 1)) {
+                lua_pushfstring(L, "setBorderSizes: bad argument #1 value (new heights size as number expected, got %s!)", luaL_typename(L, 1));
+                return lua_error(L);
+            } 
+            if (!lua_isnumber(L, 2)) {
+                lua_pushfstring(L, "setBorderSizes: bad argument #2 value (new widths size as number expected, got %s!)", luaL_typename(L, 2));
+                return lua_error(L);
+            } 
+            sizeTop = lua_tonumber(L, 1);
+            sizeRight = lua_tonumber(L, 2);
+            sizeBottom = lua_tonumber(L, 1);
+            sizeLeft = lua_tonumber(L, 2);
+            break;
+        }
+        case 3: {
+            if (!lua_isnumber(L, 1)) {
+                lua_pushfstring(L, "setBorderSizes: bad argument #1 value (new top size as number expected, got %s!)", luaL_typename(L, 1));
+                return lua_error(L);
+            } 
+            if (!lua_isnumber(L, 2)) {
+                lua_pushfstring(L, "setBorderSizes: bad argument #2 value (new widths size as number expected, got %s!)", luaL_typename(L, 2));
+                return lua_error(L);
+            } 
+            if (!lua_isnumber(L, 3)) {
+                lua_pushfstring(L, "setBorderSizes: bad argument #3 value (new bottom size as number expected, got %s!)", luaL_typename(L, 3));
+                return lua_error(L);
+            } 
+            sizeTop = lua_tonumber(L, 1);
+            sizeRight = lua_tonumber(L, 2);
+            sizeBottom = lua_tonumber(L, 3);
+            sizeLeft = lua_tonumber(L, 2);
+            break;
+        }
+        case 4: {
+            if (!lua_isnumber(L, 1)) {
+                lua_pushfstring(L, "setBorderSizes: bad argument #1 value (new top size as number expected, got %s!)", luaL_typename(L, 1));
+                return lua_error(L);
+            } 
+            if (!lua_isnumber(L, 2)) {
+                lua_pushfstring(L, "setBorderSizes: bad argument #2 value (new right size as number expected, got %s!)", luaL_typename(L, 2));
+                return lua_error(L);
+            } 
+            if (!lua_isnumber(L, 3)) {
+                lua_pushfstring(L, "setBorderSizes: bad argument #3 value (new bottom size as number expected, got %s!)", luaL_typename(L, 3));
+                return lua_error(L);
+            } 
+            if (!lua_isnumber(L, 4)) {
+                lua_pushfstring(L, "setBorderSizes: bad argument #4 value (new left size as number expected, got %s!)", luaL_typename(L, 4));
+                return lua_error(L);
+            } 
+            sizeTop = lua_tonumber(L, 1);
+            sizeRight = lua_tonumber(L, 2);
+            sizeBottom = lua_tonumber(L, 3);
+            sizeLeft = lua_tonumber(L, 4);
+            break;
+        }
+        default: {
+            lua_pushfstring(L, "setBorderSizes: wrong number of arguments (0-4 expected, got %d!)", numberOfArguments);
```

Comment:
It shouldn't be an error to give more than needed - all Lua functions allow this.